src/content/self-hosted/helm-configuration.mdx

```diff
@@ -269,6 +290,9 @@ The daemonset automatically configures every node of your cluster to work better
 - `extraEnv`: Environment variables to be set on the daemonset containers.
 - `labels`: Labels to add to the daemonset pods.
 - `image`: Container image used by the daemonset pods.
+  - `registry`: The registry where the daemonset image is hosted. Defaults to `` (You can change all the registries by setting `globals.registry` to your registry).
+  - `repository`: The repository path for the daemonset image. Defaults to `okteto/daemon` if not specified.
+  - `tag`: The image tag to deploy. Defaults to `0.14.0` if not specified.
```

Comment:
Same here, I thought we were tagging all our images with the chart release on every release to simplify the airgap configuration